docs/authentication/social-connections/discord.mdx

```diff
@@ -1,38 +1,79 @@
 ---
 title: Add Discord as a social connection
-description: Learn how to set up social connection with Discord.
+description: Learn how to allow users to sign up and sign in to your Clerk app with their Discord account.
 ---
 
-How to set up social connection with Discord
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "A Clerk app is required.",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "A Discord account is required.",
+      link: "https://discord.com/register",
+      icon: "user-circle",
+    }
+  ]}
+>
+  - Use Discord to authenticate users with OAuth
+</TutorialHero>
 
-## Overview
+Enabling OAuth with [**Discord**](https://discord.com/developers/docs/topics/oauth2) allows your users to sign up and sign in to your Clerk application with their Discord account.
 
-Adding social connection with Discord to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+## Configure for your development instance
 
-To make the development flow as smooth as possible, Clerk uses preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+For _development instances_, Clerk uses preconfigured shared OAuth credentials and redirect URIs—no other configuration is needed.
 
-For production instances, you will need to generate your own Client ID and Client secret using your Discord account.
+1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+1. Select **Add connection** and select **For all users**.
+1. In the **Choose provider** dropdown, select **Discord**.
+1. Select **Add connection**.
 
-> [!NOTE]
-> The purpose of this guide is to help you create a Discord account and a Discord OAuth app - if you're looking for step-by-step instructions using Clerk to add social connection (OAuth) to your application, follow the [Social connection (OAuth) guide](/docs/authentication/social-connections/oauth).
+## Configure for your production instance
 
-## Before you start
+In _production instances_, you must provide custom credentials, which involves generating your own **Client ID** and **Client Secret** using your Microsoft Entra ID account.
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a Discord account. To create one, [click here](https://discord.com/register).
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Discord Developers](https://discord.com/developers/applications) page.
```

Comment:
```suggestion
To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Discord Developers Portal](https://discord.com/developers/applications).
```

We can call it what they call it!
<img width="559" alt="Screenshot 2024-10-09 at 10 30 28" src="https://github.com/user-attachments/assets/13b9bd52-d8a0-4a21-b0bb-bf1ae75ece47">